x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_export_by_object_ids.test.ts

```diff
@@ -60,6 +60,7 @@ describe('get_export_by_object_ids', () => {
         rulesNdjson: {
           author: ['Elastic'],
           actions: [],
+          active_snoozes: [],
```

Comment:
By the way, regarding rule export and import, I'm not sure we should support snooze data in export/import. It feels like too dynamic/sporadic data that can become outdated pretty quickly whereas exported data should be static and not bound to time.

Reply:
Make sense!